hw/ip_templates/rstmgr/data/rstmgr.tpldesc.hjson

```diff
@@ -129,6 +130,12 @@
       type: "bool"
       default: "1"
     }
+    {
+      name: "with_cpu"
```

Comment:
Rename to `with_cpu_dump`? In the same vein, a better name for `with_alert_handler` would be `with_alert_dump`.